functions/lib/stripe/webhooks.py

```
@@ -855,7 +1002,172 @@ def stripe_webhook_checkout_fulfilment(req: https_fn.Request) -> https_fn.Respon
             )
+        case "payment_intent.canceled":
+            logger.add_tags(
+                {
+                    "eventType": event["type"],
+                    "eventId": event.id,
+                    "paymentIntentId": event["data"]["object"]["id"],
+                    "stripeAccount": event["account"],
+                }
+            )
+            logger.info(
+                f"Processing webhook event of payment_intent.canceled for {event.id}."
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)